.github/workflows/manage-integration-check.yml

```diff
@@ -65,12 +65,23 @@ jobs:
               external_id,
             })
             core.debug(`integration-test-result check runs: ${JSON.stringify(runs, null, 2)}`);
-            const filtRuns = runs.filter(run => run.status !== 'completed');
+            const filtRuns = runs.filter(run => run.status !== 'completed' && run.external_id === external_id);
             const descRuns = filtRuns.sort((a, b) => Date.parse(b.started_at) - Date.parse(a.started_at));
             const run = descRuns[0];
 
             if (!run) {
-              core.setFailed(`No integration-test-result check found for commit ${head_sha} ${external_id}`);
+              const check = await github.rest.checks.create({
+                ...context.repo,
+                head_sha,
+                name: "integration-test-result",
+                status: "completed",
+                conclusion: context.payload.workflow_run.conclusion,
+                external_id,
+                output: {
+                  title: "Integration Test Aggregate Result",
+                  summary: `Synthetic check capturing the result of the <a href="${context.payload.workflow_run.html_url}">integration-test workflow run</a>`,
```

Comment:
Since we're now explicitly taking into consideration the attempt's number, let's include that detail in the summary, at least if it's not the 1st attempt. Here and in the normal creation place.